{
  "affected": [
    {
      "database_specific": {
        "source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2025-4225.json"
      },
      "ranges": [
        {
          "events": [
            {
              "introduced": "e4567ef4362f4ef24a09112564b31dfe9044ea12"
            },
            {
              "fixed": "37cd445f04ca9225834e84c0705b8cc8a5ef92c6"
            }
          ],
          "repo": "https://gitlab.com/gitlab-org/gitlab",
          "type": "GIT"
        }
      ],
      "versions": []
    }
  ],
  "details": "An issue has been discovered in GitLab CE/EE affecting all versions from 14.1 before 18.1.5, 18.2 before 18.2.5, and 18.3 before 18.3.1 that that under certain conditions could have allowed an unauthenticated attacker to cause a denial-of-service condition affecting all users by sending specially crafted GraphQL requests.",
  "id": "CVE-2025-4225",
  "modified": "2025-08-28T04:16:59Z",
  "published": "2025-08-27T20:15:32Z",
  "references": [
    {
      "type": "REPORT",
      "url": "https://gitlab.com/gitlab-org/gitlab/-/issues/538983"
    },
    {
      "type": "WEB",
      "url": "https://hackerone.com/reports/3100624"
    }
  ],
  "schema_version": "1.7.3"
}